tvalmetrics/rag_scores_calculator.py

```diff
@@ -151,7 +153,7 @@ def scores_to_dict(self) -> Dict[str, List[Optional[float]]]:
         )
 
 
-class RagScoresCalculator(object):
+class TonicValidateCalculator(object):
```

Comment:
I think we should keep this named `RagScoresCalculator` to prevent breaking changes with this PR.
```suggestion
class RagScoresCalculator(object):
```